scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -3321,7 +3349,11 @@ def __get_log_config_for_container(self, cid, info, k8s_cache, base_attributes):
                         # field `_k8s_ck`
                         container_attributes["_k8s_dn"] = controller.name
                         # `_k8s_dl` is translated to `k8s-labels`
-                        container_attributes["_k8s_dl"] = controller.flat_labels
```

Comment:
This is the only place flat_labels is used, recommend removing it from the Controller constructor